src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseServiceTests.cs

```diff
@@ -1429,6 +1454,87 @@ await using (var context = InMemoryApplicationDbContext(contextId))
             }
         }
 
+        [Fact]
+        public async Task GetInternalReleaseNotes()
+        {
+            var release = new Release();
+            var ignoredRelease = new Release();
+            var releaseStatuses = new List<ReleaseStatus>
+            {
+                new ReleaseStatus
+                {
+                    Release = release,
+                    InternalReleaseNote = "Note1 - old",
+                    ApprovalStatus = ReleaseApprovalStatus.Draft,
+                    Created = new DateTime(2001, 1, 1),
+                    CreatedBy = new User { Email = "note1@test.com" }
+                },
+                new ReleaseStatus
+                {
+                    Release = release,
+                    InternalReleaseNote = "Note2 - latest",
+                    ApprovalStatus = ReleaseApprovalStatus.HigherLevelReview,
+                    Created = new DateTime(2002, 2, 2),
+                    CreatedBy = new User { Email = "note2@test.com" }
+                },
+                new ReleaseStatus
+                {
+                    Release = release,
+                    InternalReleaseNote = "Note3 - null created",
+                    ApprovalStatus = ReleaseApprovalStatus.Approved,
+                    Created = null, // Migrated ReleaseStatus has null Created and CreatedBy
+                    CreatedBy = null
+                },
+                new ReleaseStatus
+                {
+                    Release = ignoredRelease,
+                    InternalReleaseNote = "Note4 - different release",
+                    ApprovalStatus = ReleaseApprovalStatus.Approved,
+                    Created = new DateTime(2012, 12, 12),
+                    CreatedBy = new User { Email = "note4@test.com" }
+                },
+            };
+
+            var contextId = Guid.NewGuid().ToString();
+            await using (var contentDbContext = InMemoryApplicationDbContext(contextId))
+            {
+                await contentDbContext.AddRangeAsync(release, ignoredRelease,
+                    releaseStatuses[0], releaseStatuses[1],
+                    releaseStatuses[2], releaseStatuses[3]);
+                await contentDbContext.SaveChangesAsync();
+            }
+
+            await using (var contentDbContext = InMemoryApplicationDbContext(contextId))
+            {
+                var releaseService = BuildReleaseService(contentDbContext);
+                var result = await releaseService.GetInternalReleaseNotes(release.Id);
+                Assert.True(result.IsRight);
+
+                var internalNotes = result.Right;
+                Assert.Equal(3, internalNotes.Count);
+
+                releaseStatuses = releaseStatuses
```

Comment:
Why not just set them up in a list within the Release in the ReleaseStatusesField, then releaseStatuses = `release.ReleaseStatuses` ad when you add all the objects to the db you only need to `contentDbContext.AddRangeAsync(release, ignoredRelease)`